Packs/GCP/Integrations/GCP/GCP.yml

```diff
@@ -0,0 +1,1603 @@
+commonfields:
+  id: GCP
+  version: -1
+sectionOrder:
+- Connect
+- Collect
+configuration:
+- displaypassword: Service Account Private Key file content (JSON).
+  name: credentials
+  required: false
+  type: 9
+  hiddenusername: true
+  section: Connect
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  section: Connect
+  required: false
+  advanced: true
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+  section: Connect
+  advanced: true
+name: GCP
+display: GCP
+category: Cloud Services
+description: Cloud integrations are installed from the Data Sources page. To configure a cloud integration, go to Settings > Data Sources and click "Add Data Source", select GCP, then in Advanced Settings > Security Capabilities, enable "Automation".
+script:
+  commands:
+  - name: gcp-compute-firewall-patch
+    description: Updates the specified firewall rule with the data included in the request.
+    arguments:
+    - name: project_id
+      required: true
+      description: GCP project ID.
+    - name: resource_name
+      required: true
+      description: Name of the firewall rule to update.
+    - description: An optional description of this resource. Provide this property when you create the resource.
+      name: description
+    - description: URL of the network resource for this firewall rule. If not specified when creating a firewall rule, the default network is used.
+      name: network
+    - description: Priority for this rule. This is an integer between 0 and 65535, both inclusive. When not specified, the value assumed is 1000. Relative priorities determine precedence of conflicting rules. Lower value of priority implies higher precedence (eg, a rule with priority 0 has higher precedence than a rule with priority 1). DENY rules take precedence over ALLOW rules having equal priority.
+      name: priority
+    - description: 'If source ranges are specified, the firewall will apply only to traffic that has source IP address in these ranges. These ranges must be expressed in CIDR format. One or both of sourceRanges and sourceTags may be set. If both properties are set, the firewall will apply to traffic that has source IP address within sourceRanges OR the source IP that belongs to a tag listed in the sourceTags property. The connection does not need to match both properties for the firewall to apply. Only IPv4 is supported. comma separated. '
```

Comment:
If source ranges are specified, the firewall applies only to traffic with source IP addresses in these ranges. These ranges must be expressed in CIDR format. One or both of sourceRanges and sourceTags may be set. If both properties are set, the firewall applies to traffic with a source IP address within sourceRanges OR the source IP belonging to a tag listed in the sourceTags property. The connection does not need to match both properties for the firewall to apply. Only IPv4 is supported, comma-separated. '